service/src/main/resources/api/swagger-ui.html

```diff
@@ -0,0 +1,37 @@
+<!-- HTML for static distribution bundle build -->
```

Comment:
I'd be interested to see if we could use a similar setup to what jade-data-repo is doing here. Or maybe that can wait for a future PR. Specifically they have a template version of this file that substitutes `clientId` and `clientSecret` when the page is rendered. I think we need something similar to allow the swagger page to be used for authenticated requests. See https://github.com/DataBiosphere/jade-data-repo/blob/develop/src/main/resources/templates/index.html